spartan/aztec-network/templates/prover-broker.yaml

```diff
@@ -85,7 +85,7 @@ spec:
             - "-c"
             - |
               source /shared/config/service-addresses
-              /scripts/configure-prover-env.sh ${BOOT_NODE_HOST}
+              /scripts/configure-prover-env.sh ${BOOT_NODE_HOST:-} ${FULL_NODE_HOST:-}
```

Comment:
Heads up that if `BOOT_NODE_HOST` is not set, then `FULL_NODE_HOST` will end up being the first arg and be interpreted as the boot node. Not sure if that's possible though.

Reply:
Yes, this is wrong. I fixed it on the `prover-node` but missed it here.